docs/examples/classify_order.md

```diff
@@ -4,7 +4,9 @@ To learn about the basics of creating an order, please refer to the [quickstart
 
 ## Order description
 
-With this order we want to find out what kind of emotions certain AI generated image convey. We have asked Dalle-3 to generate 4 different images that would convey happiness, anger, disgust and sadness. Now we want to find out if the images are actually match the emotions we asked for.
+With this order we want to find out what kind of emotions certain AI generated image convey. We have asked Dalle-3 to generate 4 different images that would convey happiness, anger, disgust and sadness and saved those images to publically accessible URL. 
+
+Now we want to find out if the images actually match the emotions we asked for. When you run this with your own examples, you may use simple, local paths to your images instead of the URLs.
```

Comment:
```suggestion
Now we want to find out if the images actually match the emotions we asked for. When you run this with your own examples, you may use local paths to your images instead of the URLs.
```